JobConfig/digitize/digitize.fcl

```diff
@@ -38,4 +38,4 @@ services.TFileService.fileName : "/dev/null"
 # test values; these get overwritten by generate_fcl
 services.SeedService.baseSeed         :  773651
 services.SeedService.maxUniqueEngines :  20
-
+services.scheduler.wantSummary        : true
```

Comment:
This adds a lot of output for production log files, I would prefer not to have this on by default.

Reply:
Sure, I can remove it. My intent was to give a quick way to check the trigger results without running any analyzer.